packages/core/src/vaults/fetchVaultActivity.ts

```diff
@@ -273,7 +273,7 @@ export const makeFetchVaultActivity = ({
     const activity: VaultActivity[] = [];
     const feeDistributorAddresses = [
       getChainConstant(NFTX_FEE_DISTRIBUTOR, network),
-      ...getChainConstant(NFTX_FEE_DISTRIBUTOR_LEGACY, network),
+      ...getChainConstant(NFTX_FEE_DISTRIBUTOR_LEGACY, network, []),
```

Comment:
Can't we just make the last param optional instead of passing in empty values?

Reply:
`getChainConstant` is a generic fn that we use all over and the return value is also generic, so if you want a fallback value, you have to pass it in (or it returns undefined).